swagger.json

```diff
@@ -4508,6 +4624,27 @@
             }
           }
         },
+        "HosCertifiedDayChartResponse": {
+            "type": "object",
+            "properties": {
+                "dayStartMs": {
+                    "type": "integer",
+                    "format": "int64",
+                    "description": "Unix epoch time (in ms) of time of when this daily log starts",
+                    "example": 1462881998034
+                },
+                "shippingDocNumbers": {
+                    "type": "string",
+                    "description": "Customer Shipping Document Numbers, potentially multiple numbers delimited by a comma or space",
+                    "example": "1059984"
+                },
+                "trailerNames": {
+                    "type": "string",
+                    "description": "Trailer Number, potentially multiple names delimited by a comma or space",
```

Comment:
nit: can we use `Trailer name` in the description rather than `Trailer Number`? This more closely aligns with the description of the patch parameter.

Reply:
done